src/main/java/edu/hm/hafner/analysis/registry/OELintAdvDescriptor.java

```diff
@@ -0,0 +1,21 @@
+package edu.hm.hafner.analysis.registry;
+
+import edu.hm.hafner.analysis.IssueParser;
+import edu.hm.hafner.analysis.parser.OELintAdvParser;
+
+/**
+ * Descriptor for oelint-adv.
+ */
+class OELintAdvDescriptor extends ParserDescriptor {
+    private static final String ID = "oelint-adv";
+    private static final String NAME = ID;
+
+    OELintAdvDescriptor() {
+        super(ID, NAME);
+    }
+
+    @Override
+    public IssueParser createParser(final Option... options) {
+        return new OELintAdvParser();
+    }
```

Comment:
There are also methods to provide a URL, help text or icon. If you have those please add them.

If not it is fine to skip them.

Reply:
Link added, there's no icon available unfortunately. Since there's nothing to take care of, I guess no help is needed.